lib/temping.rb

```diff
@@ -21,7 +21,7 @@ def teardown(clear_dependencies: false)
           end
         end
         @model_klasses.clear
-        ActiveSupport::Dependencies.clear if clear_dependencies
+        ActiveSupport::Dependencies::Reference.clear!
```

Comment:
OK. Fair enough.
Can you then remove the named argument from the method definition? As it's not used anywhere else, we may as well remove it.
This can go in a minor version bump because of the possible breaking change.